CodeEdit/Features/Breadcrumbs/Views/BreadcrumbsMenu.swift

```diff
@@ -84,6 +84,9 @@ final class BreadcrumbsMenuItem: NSMenuItem {
         )?.withSymbolConfiguration(.init(paletteColors: [NSColor(color)]))
         self.image = image
         representedObject = fileItem
+        if fileItem.isFolder == true {
```

Comment:
This can just be

```
if fileItem.isFolder {
```

since it already is a Boolean expression

Reply:
oh sorry, missed that